test/features/app/spec.js

```diff
@@ -188,7 +188,7 @@ function validateArray(val, resVal, key) {
         `Different size of array for field ${key}, expected ${arrVal.length
         } actual ${resVal.length}`);
     arrVal.forEach((item, index) => {
-        validateBody(resVal[index], item); // eslint-disable-line
```

Comment:
how confident are we that validateBody and validateArray can't sometimes end up in a recursive death spiral since they reference each other? :D

Reply:
It's entirely possible, I'm just removing the eslint-disable as it's a no-op and so in fact causes an error from eslint. :-)

Reply:
gotcha :)